frontend/src/components/Schedules/Schedules.styles.ts

```diff
@@ -1,25 +1,20 @@
 import { css } from '@emotion/react';
-import type { CSSProperties } from 'react';
 
 import theme from '@styles/theme';
 
-const COLOR_VARIANTS = {
-  1: '#FFEBE8',
-  2: '#FFD4D1',
-  3: '#FFBABC',
-  4: '#FFA9B4',
-  5: '#FF8DA6',
-  6: '#DB678B',
-  7: '#B74775',
-  8: '#932C61',
-  9: '#7A1B54',
-} as Record<number, CSSProperties['color']>;
-
 export const s_container = css`
-  overflow-x: auto;
   display: flex;
-  justify-content: flex-start;
-  width: 100%;
+  flex-direction: column;
+`;
+
+export const s_relativeContainer = css`
+  position: relative;
+`;
+
+export const s_scheduleTableContainer = css`
+  display: flex;
+  width: 90%;
+  min-height: fit-content;
 `;
```

Comment:
[P2]

 ```suggestion
export const s_scheduleTableContainer = css`
  display: flex;
  width: 90%;
  min-height: fit-content;

  span::selection {
    user-select: none;
  }
`;
```

드래그를 오른쪽에서 왼쪽으로 올릴 때 대비해서 user-select를 없애는 방법도 좋을 것 같네요 :)

Reply:
앗...이 부분에 대한 드래그는 놓쳤는데 섬세한 피드백 감사합니다! 반영하도록 하겠습니다 :)